syntax/forth.vim

```diff
@@ -1,10 +1,10 @@
 " Vim syntax file
-" Language:    FORTH
-" Current Maintainer:  Johan Kotlinski <kotlinski@gmail.com>
-" Previous Maintainer:  Christian V. J. Br�ssow <cvjb@cvjb.de>
-" Last Change: 2023-01-12
-" Filenames:   *.fs,*.ft
-" URL:	       https://github.com/jkotlinski/forth.vim
+" Language:		FORTH
+" Current Maintainer:	Johan Kotlinski <kotlinski@gmail.com>
+" Previous Maintainer:	Christian V. J. Br�ssow <cvjb@cvjb.de>
```

Comment:
If we are anyway touching this line. Do you think the spelling of "Brüssow" can be corrected? I am not sure what are the rules for non-ASCII characters in Vimscript.

Reply:
Done